### ISSUE TITLE
Version mix - Roadmap 1.4.0

### ISSUE THREAD

**Issue by fjcano-stratio** [bguerout/jongo#300]
Hello @bguerout ,

We are trying with **jongo-1.4-early-20160606-0118** , **mongo-java-driver-3.3.0.jar**, db version **v3.2.8** and some basic queries and it looks fine.. 

Do you know any restriction or limitations for this mix?

Do you know (approx) when jongo move out from early to canditate or release?

Thanks and congrats for your effort.

**Comment by bguerout** [comment]
Hello,

Sorry for the late response. The mix seems ok to me.
It should be possible to release 1.4 by the end of September or something like that

**Comment by bguerout** [comment]
i'm going to make some tests

**Comment by fjcano-stratio** [comment]
Thanks a lot @bguerout !

**Comment by hoegertn** [comment]
Any current roadmap? @bguerout

**Comment by bguerout** [comment]
Hello,

To be true the 1.4 version is almost done since few months (June 2016) and afaik only the documentation (ie. website) needs to be updated.

I'm not totally satisfied with v3 API and its test coverage  (https://github.com/bguerout/jongo/tree/master/src/test/java/org/jongo/v3).
It is very hard for me to find time to work on Jongo but i will do my best to finish what has been started.

Note that you can still use the last beta version 

``` xml
<dependencies>
  <dependency>
    <groupId>org.jongo</groupId>
    <artifactId>jongo</artifactId>
    <version>jongo-1.4-early-20161223-1142</version>
  </dependency>
</dependencies>
...
<repositories>
    <repository>
       <id>cloudbees-jongo-early-release</id>
        <url>http://repository-jongo.forge.cloudbees.com/release</url>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>false</enabled>
        </snapshots>
    </repository>
</repositories>
```

**Comment by bguerout** [comment]
Closed in favor of #254 
Feel free to reopen if needed

**Comment by bguerout** [comment]
Closed in favor of #254
